django/db/backends/base/schema.py

```diff
@@ -215,6 +217,10 @@ def column_sql(self, model, field, include_default=False):
         # Check for fields that aren't actually columns (e.g. M2M)
         if sql is None:
             return None, None
+        # Set collation
+        collation = getattr(field, 'db_collation', None)
+        if collation:
+            sql += f' {self.connection.ops.collate_sql(collation)}'
```

Comment:
We should add a trailing space in `collate_sql()`:
```suggestion
            sql += self.collate_sql(collation)
```
and
```python
    def collate_sql(self, collation):
        return ' COLLATE ' + collation
```